ThinBridge/Chronos
# Improve message loop

# Which issue(s) this PR fixes:

https://github.com/ThinBridge/Chronos/issues/155

# What this PR does / why we need it:

Why we need this:

The redirection is slow in the some pages[1]. 
That is caused by a lack of the implementation of `OnScheduleMessagePumpWork()`.

^1: I can't provide the site to reproduce because it is the internal site for a company.

What this PR does:

* Implement `OnScheduleMessagePumpWork()`
* Add the `MessageLoopWorker` class to manage the timer for `CefDoMessageLoopWork()`
  * `MessageLoopWorker` works on a special message window.
* Remove `Sazabi::PumpMessage()` because it is no longer needed.
* Move process to read leftover messages to `MessageLoopWorker`  from `CSazabi`

`MessageLoopWorker` does:

* Manage the timer so that `DoMessageLoopWork()` executed at at least 30fps.
* For `WM_SCHEDULE_CEF_WORK` message:
  * If `delayMs` is less than or equal to 0, execute `DoMessageLoopWork()` immediately. The timer does not reset.
  * If `delayMs` is greater than 0, set the timer for the specified time. Existing timers will be reset.
* For time timeout message:
  * Reset the timer and execute `DoMessageLoopWork()`.

# How to verify the fixed issue:

## The steps to verify:

1. Open the site with slow redirection.
2. Check if other sites work fine.
    * Full testing will be performed as an integration test. Check some cases at this PR.
3. Check if the Chronos components like the menu bar or the setting dialog and so on work fine.
    * Full testing will be performed as an integration test. Check some cases at this PR.
4. Check if shortcut keys like Ctrl + N wrok fine.
    * Full testing will be performed as an integration test. Check some cases at this PR.

## Expected result:

> 1. Open the site with slow redirection.

The redirection is as fast as v13.1.112.0.

> 2. Check if other sites work fine.

They work fine.

> 3. Check if the Chronos components like the menu bar or the setting dialog and so on work fine.

They work fine.

> 4. Check if shortcut keys like Ctrl + N work fine.

They work fine.
